tools/shared/pragma.c

```diff
@@ -647,10 +648,34 @@ do_sw(void)
       bclr(DIR_OFFSET(currdir, x[19]), 0x40);
     break;
   case SW_VECTOR:
-    if (no_specified)
+    if (no_specified) {
       bset(DIR_OFFSET(currdir, x[19]), 0x18); /* notransform | norecog */
-    else
-      bclr(DIR_OFFSET(currdir, x[19]), 0x18);
+      bclr(DIR_OFFSET(currdir, x[19]), 0x400);
```

Comment:
This code seems to be repurposing `XBIT(19, 0x400)`. Could you please update `tools/flang2/docs/xflag.n`?

Reply:
Done.